tests/components/fritz/test_update.py

```diff
@@ -47,6 +49,7 @@ async def test_update_entities_initialized(
     assert len(updates) == 1
 
 
+@pytest.mark.xfail(reason="Flaky test")
```

Comment:
What will be output if the test passes?

Reply:
I found the answer in the `strict` parameter docs for xfail.

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-xfail